images/win/scripts/Installers/Install-Git.ps1

```diff
@@ -14,11 +14,16 @@ Choco-Install -PackageName hub
 
 # Add to PATH
 Add-MachinePathItem "C:\Program Files\Git\bin"
+$env:Path = Get-MachinePath
 
 if (Test-IsWin16) {
     $env:Path += ";$env:ProgramFiles\Git\usr\bin\"
 }
 
+# Set core.longpaths option to true in the system-wide [path]/etc/gitconfig file
+# https://github.com/git-for-windows/git/blob/main/Documentation/config/core.txt
+git config --system core.longpaths true
```

Comment:
I _hope_ that this feature is stable enough to enable as widely as on the build agents... Be prepared to roll back on short notice, it might break users' scenarios in surprising and dramatic ways.

Reply:
@dscho, It's better to be safe.